src/main/java/org/snakeyaml/engine/v2/exceptions/ComposerException.kt

```diff
@@ -24,10 +24,10 @@ import java.util.*
  * @param problemMark - the problem location
  */
 class ComposerException @JvmOverloads constructor(
-    context: String = "",
-    contextMark: Optional<Mark> = Optional.empty<Mark>(),
     problem: String?,
     problemMark: Optional<Mark>,
+    context: String = "",
+    contextMark: Optional<Mark> = Optional.empty<Mark>(),
 ) : MarkedYamlEngineException(
```

Comment:
re-ordered to put the optional parameters last, which works better during usage.